tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2019_04_17/threads_compute_hashes.pysnap

```diff
@@ -1,15 +1,15 @@
 ---
-created: '2020-08-25T09:17:12.844476Z'
+created: '2021-02-22T14:11:57.242869Z'
 creator: sentry
 source: tests/sentry/grouping/test_variants.py
 ---
 app:
   hash: null
   component:
     app (threads of system take precedence)
-      threads (ignored because this variant does not have a contributing stacktrace, but the system variant does)
-        stacktrace
-          frame (non app frame)
+      threads (ignored because hash matches system variant)
```

Comment:
this comes from my fix, however without armin's commits my fixes don't change snapshots at all